css-three-column-layout/styles.css

```diff
@@ -0,0 +1,46 @@
+.container {
+  width: 100%;
+  height: 800px;
+  background-color: rgb(26, 150, 218);
```

Comment:
You probably still want to put the `background-color` on the `body`, so it fills the whole viewport.